source/detail/serialization/xlsx_producer.cpp

```diff
@@ -2310,6 +2310,8 @@ void xlsx_producer::write_worksheet(const relationship &rel)
         const auto wb_view = source_.view();
         const auto view = ws.view();
 
+        write_attribute("showGridLines", write_bool(view.show_grid_lines()));
```

Comment:
Nice catch! I just checked Excel and this attribute is only written when grid lines are disabled and enabled is the default state. In order to maintain round-trip compatibility, would you mind wrapping this in an if statement like this?
```c++
if (!view.show_grid_lines())
{
    write_attribute("showGridLines", write_bool(view.show_grid_lines()));
}
```